apps/site/lib/site_web/views/partial/expandable_block.ex

```diff
@@ -0,0 +1,7 @@
+defmodule SiteWeb.PartialView.ExpandableBlock do
```

Comment:
I did that initially but then I created the separate module in case it could be reused in the future. However it's so small that we could just use `SiteWeb.PartialView.render` indeed, which is more straightforward. 👍